src/controllers/outlook.cr

```diff
@@ -0,0 +1,15 @@
+class Outlook < ActionController::Base
+  base "/api/staff/v1/outlook"
+
+  get("/manifest.xml") do
+    manifest = OutlookManifest.new(
+      app_domain: "#{App::PLACE_URI}/outlook/",
```

Comment:
we don't want to use `App::PLACE_URI` this is for internal comms between the containers
we need to use the current `tenant.domain` and only if `tenant.platform == "office365"`

Reply:
Ok, can we get the tenant domain and platform without auth?
There is a requirement for the xml manifest to be served without auth.

Reply:
I guess we could make the tenant part of the path, or a query param

Reply:
it doesn't need auth, just can't use the helper directly

```crystal
domain_host = request.hostname.as(String)
tenant = Tenant.query.find { domain == domain_host }
```